config/locations.yml

```diff
@@ -151,117 +140,99 @@ justiceca:
 keyano:   
   name: "Keyano College"
   locationurl: "http://www.keyano.ca/Services/Library​"
-king:   
-  name: "The King's University"
+kings:
+  name: "The King's University College"
```

Comment:
Looks like Discovery might have this wrong.  "The King's University" is on the webpage.

Reply:
Yes it's "The King's University"